src/app/templates/report.ftl

```diff
@@ -0,0 +1,33 @@
+<#-- @ftlvariable name="report" type="org.vaccineimpact.orderlyweb.models.Report" -->
+<#-- @ftlvariable name="reportJson" type="String" -->
+<@layoutwide>
+    <#macro styles>
+        <link rel="stylesheet" href="/css/report-page.min.css"/>
+    </#macro>
+    <div class="row">
+        <div class="col-12 col-md-4 col-xl-3 sidebar">
+            <ul class="list-unstyled mb-0">
+                <li class="nav-item">
+                    <a class="nav-link active" data-toggle="tab" href="#report" role="tab">Report</a>
+                </li>
+                <li class="nav-item">
+                    <a class="nav-link" data-toggle="tab" href="#downloads" role="tab">Downloads</a>
+                </li>
+            </ul>
+            <hr/>
+        </div>
+        <div class="col-12 col-md-8 col-xl-9 tab-content">
+            <div class="tab-pane active" role="tabpanel" id="report">
+                <#include "partials/report.ftl">
```

Comment:
Maybe confusing to the have the full template and partial file both called report.ftl? Could be report details?